tests/run-tests

```diff
@@ -196,7 +201,7 @@ def run_tests(pyb, tests, args):
 
     # Some tests use unsupported features on Windows
     if os.name == 'nt':
-        skip_tests.add('import\\import_file.py') #works but CPython prints forward slashes
+        skip_tests.add('import/import_file.py') #works but CPython prints forward slashes
```

Comment:
yes: using python, module.__file__ on windows prints path\to\file whereas uPy uses path/to/file